index.html

```diff
@@ -34,44 +34,20 @@
 <!-- Headline -->
 
 <section    id="headline"> 
-
```

Comment:
![image](https://github.com/Xghayor/Portfolio/assets/98453979/8138b346-f1a8-4f61-8ac6-1fb013de766d)

- [ ] Kindly wrap your toolbar section with a `<header>` tag instead of a `<section>` tag. Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.

- [ ] Kindly wrap your hamburger button with a `<nav>` tag instead of a `<div>` tag. This is important for semantic reasons
- [ ] Kindly use `<ul>` to group the nav items and  `<li>` to list the nav items. This is important for Semantic Structure, Using semantic HTML tags like <ul> and <li> helps in creating a clear and logical structure for your web content. It enhances the accessibility of your webpage, making it easier for both humans and assistive technologies like screen readers to understand and navigate the content